### PR TITLE
jack 0.125.0

### PR DESCRIPTION
**Pull request by zmwangx**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [ ] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

"Description shouldn't include the formula name": This is a recursive acronym.

### CODE REVIEW

**Comment by zmwangx** [comment]
Note to self (or anyone who merge this): remember to bottle dependents

```
darkice
dirt
ltc-tools
moc
```

for Sierra and tick them off in #5488.

File: Formula/jack.rb [hunk 1]
```diff
@@ -24,8 +24,16 @@ class Jack < Formula
   depends_on "libsndfile"
   depends_on "libsamplerate"
 
-  # Change pThread header include from CarbonCore
-  patch :p0, :DATA if MacOS.version >= :mountain_lion
+  def install
+    # Makefile hardcodes Carbon header location
+    inreplace Dir["drivers/coreaudio/Makefile.{am,in}"],
+      "/System/Library/Frameworks/Carbon.framework/Headers/Carbon.h",
+      "#{MacOS.sdk_path}/System/Library/Frameworks/Carbon.framework/Headers/Carbon.h"
```
Comment (MikeMcQuaid) [inline Formula/jack.rb]: Can you submit this upstream?

Comment (MikeMcQuaid) [inline Formula/jack.rb]: Ah, I see it was already there. Would still be good to do so at some point but doesn't need to block this PR.

**Comment by MikeMcQuaid** [comment]
Thanks @zmwangx.

**Comment by MikeMcQuaid** [comment]
> darkice
> dirt
> ltc-tools
> moc

Started in https://bot.brew.sh/job/Homebrew%20Sierra%20Testing/1070/.

**Comment by zmwangx** [comment]
> Started in https://bot.brew.sh/job/Homebrew%20Sierra%20Testing/1070/.

Thanks.

**Pull request closed without merging**